app/controllers/super_admin/images_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Admin
+  class ImagesController < BaseController
```

Comment:
I'm not sure this does anything useful. Can we remove it and the route, and check uploading images still works?

Reply:
CRUD works without the routes/controller in there.